src/FFMpeg/Media/Video.php

```diff
@@ -129,6 +129,56 @@ public function save(FormatInterface $format, $outputPathfile)
             }
         }
 
+        // Merge Filters into one command
+        $videoFilterVars = $videoFilterProcesses = [];
+        for($i=0;$i<count($commands);$i++) {
+            $command = $commands[$i];
+            if ( $command == '-vf' ) {
+                $commandSplits = explode(";", $commands[$i + 1]);
+                if ( count($commandSplits) == 1 ) {
+                    $commandSplit = $commandSplits[0];
+                    $command = trim($commandSplit);
+                    if ( preg_match("/^\[in\](.*?)\[out\]$/is", $command, $match) ) {
+                        $videoFilterProcesses[] = $match[1];
+                    } else {
+                        $videoFilterProcesses[] = $command;   
+                    }
+                } else {
+                    foreach($commandSplits as $commandSplit) {
+                        $command = trim($commandSplit);
+                        if ( preg_match("/^\[[^\]]+\](.*?)\[[^\]]+\]$/is", $command, $match) ) {
+                            $videoFilterProcesses[] = $match[1];
+                        } else {
+                            $videoFilterVars[] = $command;
+                        }                
+                    }
+                }
+                unset($commands[$i]);
+                unset($commands[$i + 1]);
+                $i++;
+            }
+        }
+        $videoFilterCommands = $videoFilterVars;
+        $lastInput = 'in';
+        foreach($videoFilterProcesses as $i => $process) {
+            $command = '[' . $lastInput .']';
+            $command .= $process;
+            $lastInput = 'p' . $i;
+            if ( $i == count($videoFilterProcesses) - 1 ) {
+                $command .= '[out]';
+            } else {
+                $command .= '[' . $lastInput . ']';
+            }
+            
+            $videoFilterCommands[] = $command;
+        }
+        $videoFilterCommand = implode(";", $videoFilterCommands);
+        
+        if ( $videoFilterCommand ) {
+            $commands[] = '-vf';
+            $commands[] = $videoFilterCommand;
+        }
```

Comment:
the desired command is not just that simple.
because we need to scale first and then adding watermark.
the desired command is 
`ffmpeg -i input.avi -vf "movie=watermark.png [watermark]; [in][watermark] overlay=100:100 [p1]; [p1] scale=320:240 [out]" output.avi`
Also, because the pull request is to support multiple filters with ordering.
the desired command can be more complex as follow:
`ffmpeg -i input.avi -vf "movie=watermark.png [watermark]; [in][watermark] overlay=100:100 [p1]; [p1] scale=320:240 [p2]; [p2] [watermark] overlay=200:200 [p3]; [p3] scale=100:100 [out]" output.avi`